lib/JSON/Validator.pm

```diff
@@ -745,7 +745,7 @@ sub _validate_format {
   my $code = $self->formats->{$schema->{format}};
   return do { warn "Format rule for '$schema->{format}' is missing"; return }
     unless $code;
-  return unless my $err = $code->($value);
+  return unless my $err = $code->($value,$schema);
```

Comment:
I think I want this to change after all to `($value, $schema, $path)` instead of just `($value, $schema)`. This makes the callback receive all the local information that `_validate_format()` gets. The order is different though, but I think the most common case would be to need $schema and not $path.

What do you think?

Also, this line is not run through perltidy. I think you can now run the command below after 316bf1832dacdb686d0fe81ba20c87d390c950fc.

```
perltidy -b lib/JSON/Validator.pm
```